controllers/pvc.go

```diff
@@ -143,6 +156,11 @@ func (r *VolumeSnapshotBackupReconciler) buildPVCClone(pvcClone *corev1.Persiste
 		}
 
 		pvcClone.Spec.Resources = sourcePVC.Spec.Resources
+
+		// use the clonedPVCSize that is computed earlier
+		if clonedPVCSize != sourcePVCRequestSize {
```

Comment:
Why not just always set to clonedSize?

Reply:
@shawn-hurley will cloned size ever be smaller than source PVC size? This seems safer for "just in case the (almost) impossible happens"

Reply:
no right? https://github.com/migtools/volume-snapshot-mover/pull/251/files#diff-f62131094710ef63a76f5e484071a4e7254ff8a2d176f0957e2dd677495eff04R117-R120

We are just doing the same logic twice I think

Reply:
@sseago @shawn-hurley updated the PR, PTAL, Thanks !